Autopilot/CMakeLists.txt

```diff
@@ -35,8 +36,8 @@ if (${KIND_OF_BUILD} STREQUAL "FOR_TARGET")
   add_subdirectory(Libraries/STM32F7xx_HAL_Driver)
   add_subdirectory(Libraries/FreeRTOS)
 
-  file(GLOB_RECURSE C_SOURCES ../Common/Src/*.c ../Common/Src/stm32/*.c "Src/*.c" AttitudeManager/*.c Libraries/Drivers/Src/*.c)
-  file(GLOB_RECURSE CXX_SOURCES ../Common/Src/*.cpp ../Common/Src/stm32/*.cpp "Src/*.cpp" AttitudeManager/*.cpp Libraries/Drivers/Src/*.cpp)
+  file(GLOB_RECURSE C_SOURCES ../Common/Src/*.c ../Common/Src/stm32/*.c "Src/*.c" AttitudeManager/*.c PathManager/*.c Libraries/Drivers/Src/*.c)
```

Comment:
This stuff was actually added into devel when we merged waypoint manager. You should fetch the latest devel and rebase your changes on top of it.